### ISSUE TITLE
Add GammaRegressor

### ISSUE THREAD

**Issue by Thilakraj1998** [blobcity/autoai#68]
Add GammaRegressor model into the library.

Primary File to Change: https://github.com/blobcity/autoai/blob/main/blobcity/config/regressor_config.py

Reference GammaRegressor Implementation:
https://github.com/blobcity/ai-seed/blob/main/Regression/Linear%20Models/GammaRegressor.ipynb

Official API Reference for Parameter:
https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.GammaRegressor.html

Dependencies if any, must be appropriately added. Test run of the train function on a regression problem must pass, and the function must attempt to train an GammaRegressor as a potential best fit model.
